.github/workflows/test_suite.yml

```diff
@@ -9,18 +9,20 @@ on:
       - '**.py'
       - '**.geo'
       - '.github/workflows/test_suite.yml'
+      - 'pyproject.toml'
 
   # Run test suite whenever commits are pushed to an open PR
   pull_request:
     paths:
       - '**.py'
       - '**.geo'
       - '.github/workflows/test_suite.yml'
+      - 'pyproject.toml'
 
   # Run test suite every Sunday at 1AM
   schedule:
     - cron: '0 1 * * 0'
 
 jobs:
   test_suite:
-    uses: mesh-adaptation/docs/.github/workflows/reusable_test_suite.yml@main
+    uses: mesh-adaptation/docs/.github/workflows/reusable_test_suite.yml@workflow_install_test
```

Comment:
Change back to main branch after https://github.com/mesh-adaptation/docs/pull/98 merges